mediaqueries-5/Overview.bs

```diff
@@ -148,6 +148,52 @@ Detecting the ambient light level: the 'light-level' feature</h3>
 		</pre>
 	</div>
 
+<h3 id="reality">
+Detecting the display technology: the 'reality' feature</h3>
+
+<pre class='descdef mq'>
+Name: reality
```

Comment:
`reality` doesn't seem to work for me out of context. When paired with `virtual`, then it is clear what this is about, but otherwise, saying  "we now have a media query to test reality" just sounds weird.

Reply:
There's mixed reality and virtual reality. I'm unsure what 2D should be. "None"?